src/web/routes/auth/index.ts

```diff
@@ -48,26 +96,3 @@ authRouter.get(
 			});
 	},
 );
-
-/**
- * Checks whether the given Atlassian user is authorized to call Figma API.
- *
- * TODO: Replace with a link to public documentation.
- * @see https://hello.atlassian.net/wiki/spaces/MDT/pages/2796005496/RFC+-+Extending+generic+containers+specification+for+entity+associations#New-Provider-Actions
- */
-authRouter.get(
-	'/check3LO',
```

Comment:
Sorry I missed this earlier. We should keep the old route around so the app continues to work using the old action until all dependant PRs are released.

We can remove the old route as a follow up after the new action has been verified.

Reply:
You can use the new use case with the old route as the responses are compatible.

Reply:
If I understand correctly, a Connect App descriptor serves as an additional level an abstraction, which allows to specify any endpoint for an action:

```
checkAuth: {
	templateUrl: `${
		getConfig().app.baseUrl
	}/auth/checkAuth?userId={userId}`,
},
grant3LO: {
	templateUrl: `...`,
},
check3LO: {
	templateUrl: `${
		getConfig().app.baseUrl
	}/auth/checkAuth?userId={userId}`, // `check3LO` points to `checkAuth` since  its response is backwards compatible.
},
```

Therefore, as long as we keep the `check3LO` action in the app descriptor and the action points out to the backwards compatible endpoint, we are on the safe side.

Please, let me know if I am missing something.

Reply:
You're right. That seems like a good approach except I realised old installations will not have the new descriptor so to avoid breaking them when the app backend deploys to staging, it must still serve the old routes.

Reply:
Oh, you are right, Sam! Thank you for the explanation and a great catch.

✅  `check3LO` has been returned.